Docgen/business/plugins/org.polarsys.kitalpha.doc.gen.business.core/src/org/polarsys/kitalpha/doc/gen/business/core/util/LabelProviderHelper.java

```diff
@@ -39,10 +39,7 @@
 	private static final String ICON_FOLDER_NAME = "icon";
 	private static final NullProgressMonitor MONITOR = new NullProgressMonitor();
 
-	/**
-	 * Hidden consctructor
-	 */
-	private LabelProviderHelper(){
+	protected LabelProviderHelper(){
```

Comment:
I think sonar will detect an issue if there is no code in this method!! You can add a Java comment to avoid that situation.

Reply:
Sure, will do